### ISSUE TITLE
IE still can't expand metadata sections with keyboard

### ISSUE THREAD

**Issue by dougmet** [datasciencecampus/sdg-indicators#1950]
On IE11 (latest) if I tab down to the metadata sections (such as national) and hit enter, it doesn't expand.

No real errors on console. There are some funny messages about unmatched tags that I cannot make sense of.

**Comment by gingerbenw** [comment]
Closed the PR myself because I'm a maverick renegade.

**Comment by dougmet** [comment]
IE is the gift that keeps giving. There are some pages where this is still misbehaving.

**Comment by dougmet** [comment]
On hold as this may be fixed by #2022

**Comment by AnnCorp** [comment]
No longer an issue - no sections within metadata now have separated out onto different tabs
